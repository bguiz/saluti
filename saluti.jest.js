'use strict';

describe('[saluti]', () => {

  it('should link to submodules', () => {
    const saluti = require('./saluti.js');
    const moduleNames = Object.keys(saluti).sort();
    expect(moduleNames).toMatchSnapshot();
  });

});
