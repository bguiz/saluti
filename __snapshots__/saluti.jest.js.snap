// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[saluti] should link to submodules 1`] = `
[
  "general",
  "password"
]
`;
